### ISSUE TITLE
[Feature Request]: Twitch functionalities

### ISSUE THREAD

**Issue by BluesBroBo** [th-ch/youtube-music#2043]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

The idea is to use the app as a tool for background music in streams or for songrequests. 

### Proposed Solution

One would be able to connect a bot account to the app and join a twitch channel with it. Commands for information and control via chat should be available and can be (de)activated separatly for viewers, subs, vips, mods or broadcaster and message text should be editable. Usefull commands would be:

> Commands to start/pause/skip/go back to a song
> Commands to swap/shuffle playlists
> Commands to activate/deactivate shuffle
> Command to vote for a skip of the song and skip as soon as a set threshold is reached
> Info Commands for the current playing song + link, the current playlist + link, the next upcomming songs
> Automatic chat messages when playlist is swaped or a new song is playing

### Alternatives Considered

Instead of implementing that with a own bot, a general bot for that could be created that can join a channel.

**Comment by JellyBrick** [comment]
Blocked by #1909

**Comment by CodeWithMa** [comment]
> Blocked by [#1909](https://github.com/th-ch/youtube-music/pull/1909)

This seems to be resolved. Is this issue still a wontfix or just waiting for someone to implement it?

**Comment by ArjixWasTaken** [comment]
I'd assume the latter.

Although, it doesn't have to be part of th-ch/youtube-music, the API implemented by #1909 allows third-party apps to manipulate the player.

**Comment by CodeWithMa** [comment]
> Although, it doesn't have to be part of th-ch/youtube-music, the API implemented by [#1909](https://github.com/th-ch/youtube-music/pull/1909) allows third-party apps to manipulate the player.

Great Idea I'll look into writing my own app that uses the api :)

**Comment by vrbober** [comment]
We actually already did this: https://extensions.streamer.bot/t/song-request-system-youtube-music/2743

**Comment by SomeAspy** [comment]
resolved via the usage of API
